api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -127,6 +131,10 @@ message Transformation {
     // Configuration for an externally implemented transformer.
     envoy.config.core.v3.TypedExtensionConfig transformer_config = 3;
   }
+
+  // If set to true, the filter will log the request/response body and headers before
+  // this transformation is applied.
+  bool log_request_response_info = 4;
```

Comment:
this overrides any parent config, yes? can we document that?

Reply:
Not quite -- The current conditional that I'm using here https://github.com/solo-io/envoy-gloo/blob/1bfb840c09c77fe810936e557e53f1cb4c54db44/source/extensions/filters/http/transformation/transformation_logger.h#L17-L20

basically boils down to 
```
if parent_setting || child_setting
```

|parent setting enabled|child setting enabled|logs emitted?|
|----------------------|----------------------|-------------|
|false|false|false|
|false|true|true|
|true|false|true|
|true|true|true|

The idea being that there's a filter-level toggle if you're trying to debug issues with a series of transformations, or a transformation-level toggle if you're trying to debug a single transformation -- how do you feel about this? I'm happy to change this behavior if you think there's a better alternative